crypto/evp/pbe_scrypt.c

```diff
@@ -230,8 +230,10 @@ int EVP_PBE_scrypt(const char *pass, size_t passlen,
         return 1;
 
     B = OPENSSL_malloc((size_t)(Blen + Vlen));
-    if (B == NULL)
+    if (B == NULL) {
+        EVPerr(EVP_F_EVP_PBE_SCRYPT, ERR_R_MALLOC_FAILURE);
```

Comment:
when you are here, could you look at the other "return 0" w/o EVPerr ?

Reply:
Fixed !